seed_real_data.py

```
@@ -0,0 +1,213 @@
+        s_skip = 0
+        for s in settlement_rows:
+            char = name_to_char.get(s["name"])
+            if char is None:
+                print(f"[WARN]  Settlement 건너뜀 (미매칭: {s['name']})")
+                s_skip += 1
+                continue
+
+            # 이미지명에 쉼표가 있는 경우(복수 이미지) → 첫 번째만 처리
+            img_stems = [x.strip() for x in s["img_name"].split(",")] if s["img_name"] else []
+
+            for i, stem in enumerate(img_stems):
```

Ruff (0.15.2)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)